pkg/actors/reminder_track.go

```diff
@@ -7,5 +7,6 @@ package actors
 
 // ReminderTrack is a persisted object that keeps track of the last time a reminder fired.
 type ReminderTrack struct {
-	LastFiredTime string `json:"lastFiredTime"`
+	LastFiredTime  string `json:"lastFiredTime"`
+	RepetitionLeft int    `json:"repetitionLeft"`
```

Comment:
Is this backwards and forward compatible? What happens when the app is upgrading to this version and some instances of daprd are running with the old version and others with the new one? The fact that this is persisted creates concerns about compatibility.

Reply:
For backwards compatibility:
I am setting `RepetitionLeft` to `-1` in `actors.getReminderTrack` so if a reminder without `repetitionLeft` comes up it will assume as to be executed once as before.

For forwards compatibility:
I think property `repetitionLeft` will be ignored during json unmarshal.
However, there could be a problem in parsing of a period in the new ISO8601 format because old code parsed using `time.ParseDuration` which will fail.
Do you have recommendation about solving this.